pontoon/static/js/tags/admin/button.js

```diff
@@ -0,0 +1,51 @@
+
+import React from 'react';
+
+import {TagResourceManager} from './manager';
+
+
+export default class TagResourcesButton extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {open: false};
```

Comment:
Since you've enabled it, you can use an attribute for this instead, and avoid the constructor altogether. 

```javascript
export default class TagResourcesButton extends React.Component {
    state = {
        open: false,
    }

    ...
}
```

Reply:
nice!